packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -203,6 +192,45 @@ export interface EngineReportingOptions<TContext> {
    * TODO(helen): LINK TO EXAMPLE FUNCTION? e.g. a function recursively search for keys to be blocklisted
    */
   sendVariableValues?: VariableValueOptions;
+  /**
+   * This option allows configuring the behavior of request tracing and
+   * reporting to [Apollo Graph Manager](https://engine.apollographql.com/).
+   *
+   * By default, this is set to `true`, which results in *all* requests being
+   * traced and reported. This behavior can be _disabled_ by setting this option
+   * to `false`. Alternatively, it can be selectively enabled or disabled on a
+   * per-request basis using a predicate function.
+   *
+   * When specified as a predicate function, the _return value_ of its
+   * invocation (per request) will determine whether or not that request is
+   * traced and reported. The predicate function will receive the request
+   * context. If validation and parsing of the request succeeds the function will
+   * receive the request context in the
+   * [`GraphQLRequestContextDidResolveOperation](https://www.apollographql.com/docs/apollo-server/integrations/plugins/#didresolveoperation)
+   * phase)), which permits tracing based on dynamic properties, e.g., HTTP
+   * headers or the `operationName` (when available),
+   * otherwise it will receive the request context in the  [`GraphQLRequestContextDidEncounterError](https://www.apollographql.com/docs/apollo-server/integrations/plugins/#didencountererrors)
+   * phase)):
```

Comment:
Couple more syntax errors / stray parenthesis:

```suggestion
   * otherwise it will receive the request context in the  [`GraphQLRequestContextDidEncounterError`](https://www.apollographql.com/docs/apollo-server/integrations/plugins/#didencountererrors)
   * phase:
```